src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1915,7 +1915,7 @@ protected virtual void PropertySetter(BoundExpression node, BoundExpression rece
                 return false;
             }
 
-            return !Binder.AccessingAutoPropertyFromConstructor((BoundPropertyAccess)expr, _symbol);
+            return !Binder.IsPropertyAssignableFromConstructor((BoundPropertyAccess)expr, _symbol); // PROTOTYPE(semi-auto-props): Revise this method call is the behavior we want and add unit tests..
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=27&length=35)

The name doesn't really convey anything about equivalence to access the backing field. I think it would be good to reflect that in the name.